egs/wsj/s5/steps/libs/nnet3/xconfig/lstm.py

```diff
@@ -833,14 +831,16 @@ def generate_lstm_config(self):
         configs.append("# The core LSTM nonlinearity, implemented as a single component.")
         configs.append("# Input = (i_part, f_part, c_part, o_part, c_{t-1}), output = (c_t, m_t)")
         configs.append("# See cu-math.h:ComputeLstmNonlinearity() for details.")
-        configs.append("component name={0}.lstm_nonlin type=LstmNonlinearityComponent cell-dim={1} {2}".format(name, cell_dim, lstm_str))
+        configs.append("component name={0}.lstm_nonlin type=LstmNonlinearityComponent cell-dim={1} "
```

Comment:
@vijayaditya Sorry Vijay, I'm not quite clear how to use named arguments here, can you give some detailed examples?